package.json

```diff
@@ -7,14 +7,15 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
-    "test": "npm run unit",
-    "unit": "./bin/units",
+    "audit": "npm shrinkwrap; node node_modules/nsp/bin/nspCLI.js audit-shrinkwrap; rm npm-shrinkwrap.json;",
```

Comment:
there are now quite a few scripts in each module. would be great to document this somewhere in a common place to explain what each one is for.